dlt/common/libs/utils.py

```diff
@@ -38,3 +40,26 @@ def load_open_tables(
             schema_open_tables = [t for t in schema_open_tables if t in tables]
 
         return {name: client.load_open_table(table_format, name) for name in schema_open_tables}
+
+
+def is_instance_lib(obj: Any, *, class_ref: str) -> bool:
```

Comment:
this is cool! maybe a few tests would be good as we have them for most helpers. Maybe in a follow-up ticket.